minjoo/app.js

```diff
@@ -38,6 +43,44 @@ app.get("/ping", (req, res, next) => {
   res.json({ message: "pong!" });
 });
 
+//////////////////////////////////////
+// ## 2. 로그인 정보 확인 & JWT 토큰 발급 //
+//////////////////////////////////////
+app.post("/login", async (req, res) => {
+  const { email, password } = req.body;
+
+  // 1. 입력받은 email 과 매치되는 hashedPassword 를 DB 로부터 가져오기
+  const [{ hashedPassword }] = await myDataSource.query(
+    `SELECT password AS hashedPassword FROM users WHERE email = ?;
+		`,
+    [email]
+  );
```

Comment:
```suggestion
  const [{ hashedPassword }] = await myDataSource.query(`
    SELECT password AS hashedPassword 
    FROM users 
    WHERE email = ?;
  `, [email]);
```
너무 잦은 줄바꿈은 좋지 않겠지만 SQL문을 작성할 때만큼은 적절히 줄바꿈을 해주는 것이 알아보기 훨씬 좋습니다!

Reply:
이해했습니다 && Commit suggestion 하였습니다! 🫡